### DISCUSSION TITLE
Does it only support text and html type?

### DISCUSSION THREAD

**Discussion by liuweibing7** [BringingFire/rich_clipboard#7] [Q&A] [answered]
Thanks for your great work first.
I need to watch the system clipboard on Windows and show it on my application page. The flutter Clipboard only supports text type, then I found your rich_clipboard on pub, it supports html additionally, but do you have any plans to support more types, such as image, file?

**Comment by jmatth** [comment, marked as answer]
I'm planning to add support for additional types like images eventually, but haven't begun working on it. For an immediate solution, the [pasteboard][pasteboard] plugin on pub says it supports reading images and files from the clipboard.

[pasteboard]: https://pub.dev/packages/pasteboard

**Reply by liuweibing7** [reply]
Great, thanks so much. I would try the pasteboard first.
